docs/running_psalm/plugins/plugins_type_system.md

```diff
@@ -125,6 +125,7 @@ if (true === $first) {
 
 ``` php
 $a = ["a" => 1, "b" => 2]; // is ObjectLike, array{a:int(1),b:int(2)}
+$a = ["a" => null]; // is ObjectLike with optional keys/values, array{a:?int,b?:string}
```

Comment:
This is not accurate (because it’s being totally redefined). If this was in a conditional it would be true after the conditional scope was closed.

Reply:
done